### ISSUE TITLE
pip3 on Amazon Linux CodeBuild images installs a very old version of cfn-lint

### ISSUE THREAD

**Issue by RichardTribe** [aws-cloudformation/cfn-lint#2489]
### CloudFormation Lint Version

0.58.4 rather than 0.71.1

### What operating system are you using?

Image: aws/codebuild/amazonlinux2-x86_64-standard:4.0

### Describe the bug

When using cfn-lint in a CodeBuild buildspec
`- pip3 install cfn-lint`
Image: aws/codebuild/standard:6.0 gives an up-to-date 0.71.1

Image: aws/codebuild/amazonlinux2-x86_64-standard:4.0 gives 0.56.4
This is old enough that it does not accept node 16 as a lambda runtime.

Workaround: stay with ubuntu image rather than using -i E3030

(As an aside it would be cool if this great tool was built into one of these images for provsioning speed)

### Expected behavior

Get an up-to-date cfn-lint on Amazon Linux too or docs somewhere to guide.

### Reproduction template

N/A

**Comment by kddejong** [comment]
I would like to think these images are the same but I'm not able to replicate this locally.

```sh
docker run -it --entrypoint "" public.ecr.aws/codebuild/amazonlinux2-x86_64-standard:4.0 /bin/bash
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
bash-4.2# pip3 install cfn-lint
Collecting cfn-lint
   Downloading cfn_lint-0.71.1-py3-none-any.whl (2.4 MB)
```

**Comment by kddejong** [comment]
Results from my codebuild job.

<img width="864" alt="image" src="https://user-images.githubusercontent.com/36457093/202783644-389d9dc2-a90f-4366-8403-2694b4a4b016.png">

**Comment by RichardTribe** [comment]
Thanks for checking. I will repeat at my end. Sounds like it should be the same now.

**Comment by RichardTribe** [comment]
Great.
When using `Image: aws/codebuild/amazonlinux2-x86_64-standard:4.0` in CodeBuild I can now install cfn-lint **0.72.0**, so closing issue.

As a note I do see this error in the pip install:
```
Installing collected packages: pbr, jsonpointer, jsonpickle, sarif-om, networkx, junit-xml, jsonpatch, jschema-to-python, aws-sam-translator, cfn-lint
  Attempting uninstall: aws-sam-translator
    Found existing installation: aws-sam-translator 1.51.0
    Uninstalling aws-sam-translator-1.51.0:
      Successfully uninstalled aws-sam-translator-1.51.0
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
aws-sam-cli 1.58.0 requires aws-sam-translator==1.51.0, but you have aws-sam-translator 1.54.0 which is incompatible.
Successfully installed aws-sam-translator-1.54.0 cfn-lint-0.72.0 jschema-to-python-1.2.3 jsonpatch-1.32 jsonpickle-2.2.0 jsonpointer-2.3 junit-xml-1.9 networkx-2.8.8 pbr-5.11.0 sarif-om-1.0.4
```
I do not use aws-sam-cli (directly at least) so won't be effected by this I hope.
It is part of the tool set installed in this AWS image I believe.

Thanks for checking for me.
